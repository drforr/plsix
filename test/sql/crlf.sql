-- CR/LF test
CREATE FUNCTION crlf_test() RETURNS text LANGUAGE plsix
AS E'\r\n#!/bin/sh\r\necho OK\r\n';

SELECT crlf_test();
