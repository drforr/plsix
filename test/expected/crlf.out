-- CR/LF test
CREATE FUNCTION crlf_test() RETURNS text LANGUAGE plsix
AS E'\r\n#!/bin/sh\r\necho OK\r\n';
SELECT crlf_test();
 crlf_test 
-----------
 OK\r
(1 row)

